campaignion_email_to_target/src/MessageTemplateInstance.php

```diff
@@ -36,6 +36,7 @@ public function replaceTokens(array $target = NULL, Submission $submission = NUL
       $target['display_name'] = $target['salutation'];
     }
     $data['email-to-target'] = $target;
+    $data['target'] = $target;
```

Comment:
This can actually replace `$data['email-to-target']`. There is no need to keep it around.